packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -986,6 +1159,34 @@ function BudgetHeader({
         backgroundColor: colors.p5,
       }}
     >
+      {smallVPWidth ? (
```

Comment:
Having this up in the header feels a little too cramped for my taste, especially for a screen that's already super narrow. (It's also on the side of the month heading further away from the column that it affects.)

Personally, my intuition is to tap the BUDGETED/SPENT header to change it. For discoverability, maybe add a subtle indicator that would at least suggest something can be done with the header. Something like this maybe? In the order `BUDGETED - SPENT - BALANCE`, the arrow points from BUDGETED toward where SPENT would be, and vice versa. (I haven't removed the header toggle text button here, but that would go.)

![budgeted-spent](https://github.com/actualbudget/actual/assets/5862724/a9ceb04c-90c7-4f3a-812b-b6db53d85348)


CSS for the div surrounding the BUDGETED/SPENT text is roughly:
```css
// padding + negative margin to keep the column text aligned
padding: '0 8px';
margin: '0 -8px'; 
background: `linear-gradient(-45deg, ${colors.p5} 8px, transparent 0)`;
// 45deg to flip it to the lower left corner
```